### PR TITLE
Update About Us

### PR DESCRIPTION
**Pull request by calvintanwj**

### CODE REVIEW

File: docs/team/calvintanwj.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Calvin Tan's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```
Comment (NicolasCwy) [inline docs/team/calvintanwj.md:6]: Can amend this to Pocket Hotel and keep the parts relevant to you!

**Review by NicolasCwy — CHANGES_REQUESTED** [review]
lgtm! Just need to change the portfolio page

**Review by NicolasCwy — CHANGES_REQUESTED** [review]
lgtm! Just need to change the portfolio page

File: docs/team/calvintanwj.md (hunk 1 above)
Comment (calvintanwj) [reply docs/team/calvintanwj.md]: Done

**Review by NicolasCwy — APPROVED** [review]
Lgtm

File: docs/team/calvintanwj.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+**Pocket Hotel (PH)** is a specialized contact management desktop app targeted towards hotel managers who have many
+guests and staff to handle.
+
+Given below are my contributions to the project.
+
+* **Documentation**:
+    * User Guide:
+      * Added documentation for features `list` and `view`
+    * Developer Guide:
+      * Added and tweaked user stories
```
Comment (NicolasCwy) [inline docs/team/calvintanwj.md:17]: Should this be use cases?

**Review by NicolasCwy — CHANGES_REQUESTED** [review]
LGTM

**Review by NicolasCwy — APPROVED** [review]

**Pull request merged**